public/css/icinga/widgets.less

```diff
@@ -251,6 +269,10 @@ form.role-form {
     font-style: normal;
   }
 
+  .control-group .unrestricted-role {
```

Comment:
Is it important that the element being styled by this is part of a control group? I don't think so. The class is already specific enough. Please remove the selector.